### PR TITLE
fix(designing-tutorials): inline frame title

### LINKED ISSUE #1957: audit-avt1-designing-tutorials: inline frame needs title
## Detailed Description:

Inline frame does not have a 'title' attribute

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #1957 

Adds `titles` to newly added videos on `Designing -- Tutorials` page

Testing:

- [ ] Run `Accessibility Checker` & make sure the following error no longer shows:

Screen Shot 2020-11-05 at 3 15 43 PM

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

**Review by emyarod — APPROVED** [review]
should the titles be capitalized as titles as well in case it appears in a text pane? otherwise looks good to me

**Comment by andreancardona** [comment]
> should the titles be capitalized as titles as well in case it appears in a text pane? otherwise looks good to me

Ooo good question - I'll defer to @jillianhowarth on that one

File: src/pages/designing/tutorials.mdx [hunk 1]
```diff
@@ -29,7 +29,7 @@ Learn about Carbon and its history. This video covers how the IBM Design
 Language is applied to Carbon, how Carbon interacts with domain level guidance,
 and what it means to be an open source design system.
 
-<Video vimeoId="457802924" />
+<Video title="introduction to carbon" vimeoId="457802924" />
```
Comment (jillianhowarth) [inline src/pages/designing/tutorials.mdx:32]:
```suggestion
<Video title="Introduction to Carbon" vimeoId="457802924" />
```

File: src/pages/designing/tutorials.mdx [hunk 2]
```diff
@@ -42,7 +42,7 @@ This video covers how to design with components and type styles, and where to
 find relevant guidance and resources while designing. It also goes through the
 steps of getting set up with the Carbon Sketch kits and grid template.
 
-<Video vimeoId="457803206" />
+<Video title="designing with carbon" vimeoId="457803206" />
```
Comment (jillianhowarth) [inline src/pages/designing/tutorials.mdx:45]:
```suggestion
<Video title="Designing with Carbon" vimeoId="457803206" />
```

File: src/pages/designing/tutorials.mdx [hunk 3]
```diff
@@ -53,7 +53,7 @@ Learn about the fundamentals of using our 2x Grid. This video gives an overview
 of basic grid terminology, breakpoints, the three grid modes, and some resources
 to get started designing with the 2x Grid.
 
-<Video vimeoId="457802917" />
+<Video title="using 2x grid" vimeoId="457802917" />
```
Comment (jillianhowarth) [inline src/pages/designing/tutorials.mdx:56]:
```suggestion
<Video title="Using the 2x grid" vimeoId="457802917" />
```

**Review by jillianhowarth — CHANGES_REQUESTED** [review]
@andreancardona @emyarod Yep! The titles should all use sentence case capitalization

**Review by jillianhowarth — APPROVED** [review]

**Pull request merged**